test/gtest/ucp/ucp_test.cc

```diff
@@ -320,9 +319,10 @@ ucp_test_base::entity::~entity() {
 }
 
 void ucp_test_base::entity::connect(const entity* other,
-                                    const ucp_ep_params_t* ep_params_cmn) {
+                                    const ucp_ep_params_t* ep_params_cmn,
+                                    bool reuse_worker_idx) {
```

Comment:
maybe need to provide the index at which the ep would be installed in the vector? why support only same index as the worker index?